lib/slime/compiler.ex

```diff
@@ -5,27 +5,51 @@ defmodule Slime.Compiler do
 
   alias Slime.Doctype
 
-  alias Slime.Parser.Nodes.{DoctypeNode, EExNode, HTMLCommentNode, HTMLNode, InlineHTMLNode, VerbatimTextNode}
+  alias Slime.Parser.Nodes.{DoctypeNode, EExNode, HEExNode, HTMLCommentNode, HTMLNode, InlineHTMLNode, VerbatimTextNode}
+
+  alias Slime.TemplateSyntaxError
+
+  @eex_delimiters {"#" <> "{", "}"}
+  @heex_delimiters {"{", "}"}
 
   @void_elements ~w(
     area br col doctype embed hr img input link meta base param
     keygen source menuitem track wbr
   )
 
-  def compile([]), do: ""
+  def eex_delimiters, do: @eex_delimiters
+  def heex_delimiters, do: @heex_delimiters
+
+  def compile([], _delimiters), do: ""
```

Comment:
```suggestion
  def compile(x), do: compile(x, @eex_delimeters)
  def compile([], _delimiters), do: ""
```

Might (or might not!) be good to avoid breaking the public API of the module by preserving `compile/1`.

If this is done, all the changes to [test/compiler_test.exs](https://github.com/slime-lang/slime/pull/168/files#diff-d283697740dde235f1562d8332a5cd4237c3e45822890280abad8a9f3bf8e601) can be reverted.